packages/rs-drive-abci/src/abci/handlers.rs

```diff
@@ -32,720 +32,1052 @@
-                                block_height, day
-                            )
-                        });
-
-                    let after_finalize_block_request = AfterFinalizeBlockRequest {
-                        updated_data_contract_ids: Vec::new(),
-                    };
-
-                    platform
-                        .after_finalize_block(after_finalize_block_request)
-                        .unwrap_or_else(|_| {
-                            panic!(
-                                "should begin process block #{} for day #{}",
-                                block_height, day
-                            )
-                        });
-
-                    // Should pay to all proposers for epoch 0, when epochs 1 started
-                    if epoch_index != 0 && epoch_change {
-                        assert!(block_end_response.proposers_paid_count.is_some());
-                        assert!(block_end_response.paid_epoch_index.is_some());
-
-                        assert_eq!(
-                            block_end_response.proposers_paid_count.unwrap(),
-                            blocks_per_day as u16,
-                        );
-                        assert_eq!(block_end_response.paid_epoch_index.unwrap(), 0);
-                    } else {
-                        assert!(block_end_response.proposers_paid_count.is_none());
-                        assert!(block_end_response.paid_epoch_index.is_none());
-                    };
-                }
+    fn query(&self, request: RequestQuery) -> Result<ResponseQuery, ResponseException> {
+        let RequestQuery {
+            data,
+            path,
+            height,
+            prove,
+        } = request;
+
+        match self.platform.drive.query_serialized(data, path, prove) {
+            Ok(data) => {
+                Ok(ResponseQuery {
+                    //todo: right now just put GRPC error codes,
+                    //  later we will use own error codes
+                    code: 0,
+                    log: "".to_string(),
+                    info: "".to_string(),
+                    index: 0,
+                    key: vec![],
+                    value: data,
+                    proof_ops: None,
+                    height,
+                    codespace: "".to_string(),
+                })
+            }
+            Err(error) => {
+                Ok(ResponseQuery {
+                    //todo: right now just put GRPC error codes,
+                    //  later we will use own error codes
+                    code: 1,
+                    log: "".to_string(),
+                    info: "".to_string(),
+                    index: 0,
+                    key: vec![],
+                    value: vec![],
+                    proof_ops: None,
+                    height,
+                    codespace: "".to_string(),
+                })
             }
         }
     }
 }
+//
+// #[cfg(test)]
```

Comment:
I like that the tests go straight to commented))

Reply:
We have a task to deal with commented code